packages/preact/test/index.test.tsx

```diff
@@ -418,38 +505,44 @@ describe("@preact/signals", () => {
 		});
 	});
 
-	describe('hooks mixed with signals', () => {
-		it('signals should not stop context from propagating', () => {
-			const ctx = createContext({ test: 'should-not-exist' });
+	describe("hooks mixed with signals", () => {
```

Comment:
Styling change is probably unintended?

Reply:
Yes, but it's was formatted automatically, I think someone was not run prettier